mne/viz/_brain/_brain.py

```diff
@@ -2496,6 +2536,157 @@ def add_foci(self, coords, coords_as_verts=False, map_surface=None,
                                   opacity=alpha, resolution=resolution)
             self._renderer.set_camera(**views_dicts[hemi][v])
 
+    @verbose
+    def add_sensors(self, inst, trans, picks=None, colors=None,
+                    scales=None, alphas=None, meg_helmet=True,
+                    project_eeg=False, fnirs_pairs=None, fnirs_sources=None,
+                    fnirs_detectors=None, verbose=None):
+        """Add mesh objects to represent sensor positions.
+
+        Parameters
+        ----------
+        inst : mne.Info | mne.io.Raw | mne.Epochs | mne.Evoked
+            The MNE object with sensor locations.
+        %(trans_not_none)s
+        %(picks_all)s
+        colors : str | dict
+            The color of the sensors. If dict, channel names or types can be
+            keys with values of matplotlib colors. Default is all sensors
+            yellow and the MEG helmet and reference blue. The MEG helmet
+            can be colored with the key "meg_helmet".
+        scales : float | dict
+            The size of the sensors. If dict, channel names or types can be
+            keys with values for the size in mm. If None, default scales will
+            be used.
+        alphas : float | dict
+            The opacity of the sensors or, if dict, each sensor by name or
+            type. Default is all sensors 0.5 opacity. The MEG helmet
+            can be colored with the key "meg_helmet".
+        meg_helmet : bool
+            Whether to plot the MEG helmet (only applies if MEG sensors are
+            present in ``info``).
+        project_eeg : bool
+            Whether to project the EEG sensors to the scalp (only applies if
+            EEG sensors are present in ``info``).
+        fnirs_pairs : str | bool
+            The color to plot fnirs pairs. If False, fnirs pairs will not
+            be plotted. If None, the default color will be used.
+        fnirs_sources : str | bool
+            The color to plot fnirs sources. If False, fnirs sources will not
+            be plotted. If None, the default color will be used.
+        fnirs_detectors : str | bool
+            The color to plot the fnirs detectors. If False, fnirs detectors
+            will not be plotted. If None, the default color will be used.
+        %(verbose)s
+
+        Notes
+        -----
+        .. versionadded:: 0.24
+        """
+        from matplotlib.colors import colorConverter
+        if isinstance(inst, Info):
+            info = inst.copy()
+        else:
+            info = inst.info.copy()
+        picks = _picks_to_idx(info, picks, none='all', exclude=(),
+                              allow_empty=False)
+        info = info.pick_channels([inst.ch_names[idx] for idx in picks])
+        if trans is None:
+            warn('No `trans` provided, assuming identity head->mri transform')
+            trans = Transform('head', 'mri')
+        head_mri_t = _get_trans(trans, 'head', 'mri')[0]
```

Comment:
Let's not support `None` here at all. In `plot_alignment` we can be more flexible, but it's almost certainly a bug if someone has a None trans for `Brain.add_sensors` because sensors are (pretty much always) in head and Brain is always in MRI. Let's make them construct an identity one if they really want it:
```suggestion
        head_mri_t = _get_trans(trans, 'head', 'mri', allow_none=False)[0]
        del trans
```
```